### PR TITLE
Translate community.md into Chinese

### PR DESCRIPTION
**Pull request by wwwbai**
# What does this PR do?

I translated community.md into Chinese.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@stevhliu
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by wwwbai** [comment]
After three simple document translation PRs, I think I have fully understood the process for newcomers to contribute. Thank you @stevhliu@Isotr0py@blueingman for your help.If you could review this translated document , I would be particularly grateful!

**Review by stevhliu — APPROVED** [review]
Cool, thanks!

File: docs/source/zh/community.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# 社区
+
+这个页面汇集了社区开发的🤗Transformers相关的资源。
+
+## 社区资源
+
+| 资源     |      描述      |      作者      |
+|:----------|:-------------|------:|
+| [Hugging Face Transformers Glossary Flashcards](https://www.darigovresearch.com/huggingface-transformers-glossary-flashcards) | 这是一套基于 [Transformers文档术语表](glossary) 的抽认卡，它们已被整理成一种可以通过 [Anki](https://apps.ankiweb.net/) 这款专为长期知识保留而设计的开源、跨平台的应用来进行学习和复习的形式。参见这个 [介绍如何使用抽认卡的视频](https://www.youtube.com/watch?v=Dji_h7PILrw)。 | [Darigov Research](https://www.darigovresearch.com/) |
```
Comment (Isotr0py) [inline docs/source/zh/community.md:12]:
```suggestion
| [Hugging Face Transformers Glossary Flashcards](https://www.darigovresearch.com/huggingface-transformers-glossary-flashcards) | 这是一套基于 [Transformers文档术语表](glossary) 的抽认卡，它们已被整理成可以通过 [Anki](https://apps.ankiweb.net/) （一款专为长期知识保留而设计的开源、跨平台的应用）来进行学习和复习的形式。使用方法参见： [介绍如何使用抽认卡的视频](https://www.youtube.com/watch?v=Dji_h7PILrw)。 | [Darigov Research](https://www.darigovresearch.com/) |
```

**Review by Isotr0py — APPROVED** [review]
Just a nit.

**Comment by wwwbai** [comment]
> Just a nit.

Thanks a lot!!

**Pull request merged**